.github/workflows/ci.yml

```diff
@@ -6,37 +6,47 @@ jobs:
   build_and_test:
     strategy:
      matrix:
-        platform: [ubuntu-18.04]
+        platform: [ubuntu-18.04, ubuntu-20.04]
     runs-on: ${{ matrix.platform }}
+    env:
+      MG_VERSION: "1.5.0"
+
     steps:
     - name: Install system dependencies
       run: sudo apt-get install -y git cmake make gcc g++ libssl-dev
-    - name: Download mgclient
-      uses: actions/checkout@master
+
+    - uses: actions/checkout@v2
       with:
-        repository: memgraph/mgclient
+        submodules: true
+
     - name: Install mgclient
       run: |
-        mkdir build
-        cd build
-        cmake ..
+        cd mgclient
+        mkdir build && cd build
+        cmake -DCMAKE_INSTALL_PREFIX=output ..
         make
-        sudo make install
+        make install
+
+    # https://brandur.org/fragments/github-actions-env-vars-in-env-vars
+    - name: "Set environment variables"
+      run: |
+        echo "C_INCLUDE_PATH=$C_INCLUDE_PATH:$GITHUB_WORKSPACE/mgclient/build/output/include" >> $GITHUB_ENV
+        echo "LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$GITHUB_WORKSPACE/mgclient/build/output/lib" >> $GITHUB_ENV
```

Comment:
Of course, this is a problem locally, so it should be handled as part of the build somehow...

Reply:
Isn't it enough to specify this for the step that builds/tests `rsmgclient`? Similarly to `FIRST_NAME` [here](https://docs.github.com/en/actions/reference/environment-variables).

Reply:
It should be, I just think this is the most robust way in general, + this was just tmp solution to make the build pass. I've moved the whole build logic into `build.rs`, which is I think much nicer :smile:

I will resolve this because the party is inside `build.rs` now :partying_face: